### PR TITLE
add more friendly stuff

### LINKED ISSUE #1: Don't work
Hi @jbrooksuk, I type this in my CSS code:
color: asbesto;

But don't change to hex code, or where I am mistaked.
ST3 Buld 3083 (Estable)
Windows 7 x64

### PR DESCRIPTION
**Pull request by math2001**
Closes #1 
- add your plugin to the command pallete (list color)
- when you type a color name, it inserts it hex code.
- add different source (scss, sass, stylus, postcss)
- show hex code in completions popup

Matt

### CODE REVIEW

File: flatcolors.sublime-commands [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[
+	{
```
Comment (jbrooksuk) [inline flatcolors.sublime-commands]: Can you use spaces and not tabs please.

**Review by jbrooksuk — CHANGES_REQUESTED** [review]
We always use spaces.

**Comment by math2001** [comment]
Hey!
I've just commited my changes, and pused them. Do I need to do something more? I'm not used for now to this kind a stuff.

File: flatcolors.sublime-commands (hunk 1 above)
Comment (math2001) [reply flatcolors.sublime-commands]: Yes! I did a commit with the change.

**Comment by Kristinita** [comment]
@jbrooksuk , please do not forget to do [releases](https://packagecontrol.io/docs/submitting_a_package#Step_4) for your plugins, so that users of your plugins received changes. Thanks.

**Pull request merged**